tests/Model/ChangeStreamIteratorTest.php

```
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

This is necessary to disable type coercion in tests. Without this, the non-int test values `3.14` and `true` would be silently coerced to integers, which is not what we're expecting to test here. Alternatively, we can omit this `declare` and skip testing values that we know can be coerced.

---

This explanation makes sense, but let's record it in a comment above `declare()`.

---

Done here and in the other test that uses this.